gnovm/cmd/gno/testdata/doctest/1.md

```diff
@@ -0,0 +1,87 @@
+# Gno Doctest: Easy Code Execution and Testing
+
+Gno Doctest is a tool that allows you to easily execute and test code blocks written in the Gno language. This tool offers a range of features, from simple code execution to complex package imports.
+
+## 1. Basic Code Execution
+
+Even the simplest form of code block can be easily executed.
+
+```go
+package main
+
+func main() {
+    println("Hello, World!")
+}
+```
+
+Doctest also recognizes that a block of code is a gno. The code below outputs the same result as the example above.
+
+```go
+package main
+
+func main() {
+    println("Hello, World!")
+}
+```
+
+Running this code will output "Hello, World!".
+
+## 2. Using Standard Library Packages
+
+Gno Doctest automatically recognizes and imports standard library packages.
+
+```go
+package main
+
+import "std"
+
+func main() {
+    addr := std.GetOrigCaller()
+    println(addr)
+}
+```
+
+## 3. Utilizing Various Standard Libraries
+
+You can use multiple standard libraries simultaneously.
+
+```gno
+package main
+
+import "strings"
+
+func main() {
+    println(strings.ToUpper("Hello, World"))
```

Comment:
i don't get where you're putting the output?

Reply:
The execution results are not separately saved (although they are stored in the cache) and are only shown through the CLI. At first, I thought this would be sufficient, but it seems that saving the values would be more useful.